path-manager/conf/routes

```diff
@@ -5,8 +5,9 @@ GET     /paths                              controllers.PathManagerController.ge
 
 POST    /paths                              controllers.PathManagerController.registerNewPath
 PUT     /paths/:id                          controllers.PathManagerController.registerExistingPath(id: Long)
-POST    /paths/:id                          controllers.PathManagerController.updateCanonicalPath(id: Long)
+POST    /paths/:id                          controllers.PathManagerController.updateCanonicalPath(id: Long, formAlias: Boolean)
 DELETE  /paths/:id                          controllers.PathManagerController.deleteRecord(id: Long)
+DELETE  /paths                              controllers.PathManagerController.deleteAlias(path: String)
```

Comment:
given our discussions today (RE: draft vs live), do we want to remove this as an endpoint for the time being (but retain the functionality (if only for the tests)?

Reply:
Force anyone wanting to remove an alias to do it in dynamo? Probably a good way to make sure it's never misused.